apps/jubmoji-quest/src/pages/powers/[id].tsx

```diff
@@ -148,6 +148,25 @@ export default function PowerDetailPage() {
               </Link>
             }
           />
+          <PowerDetailLabel
+            label={
+              "Collect " +
+              getNumCardsToCollect(
+                power.proofType,
+                power.proofParams
+              ).toString()
+            }
+            value={power.collectionCards?.map(
+              (card) => cardPubKeys[card.index].emoji
+            )}
+          />
+          <PowerDetailLabel
```

Comment:
Do powers have a separate expiry ever? I feel like this will be confusing, down to remove

Reply:
Yeah, this would be something like a ticket for an event during the quest, but I'm happy to remove it for now